torch/_torch_docs.py

```diff
@@ -8518,9 +8518,9 @@ def merge_dicts(*dicts):
 .. note:: Irrespective of the original strides, the returned matrix `V` will
           be transposed, i.e. with strides `V.contiguous().transpose(-1, -2).stride()`.
 
-.. note:: Extra care needs to be taken when backward through outputs. Such
-          operation is really only stable when all eigenvalues are distinct.
-          Otherwise, ``NaN`` can appear as the gradients are not properly defined.
+.. warning:: Extra care needs to be taken when backward through outputs. Such
+             operation is only stable when all eigenvalues are strictly distinct and can
+             be unstable when eigenvalues are close to each other.
```

Comment:
What do you mean by `strictly distinct`? Maybe the last sentence could be  `...distinct and is more unstable the closer the eigenvalues to each other`.